vizro-core/src/vizro/static/css/layout.css

```diff
@@ -138,14 +138,6 @@
   width: 100%;
 }
 
-div.dashboard_container .tooltip-inner {
-  color: var(--text-primary);
-}
-
-div.dashboard_container .custom-tooltip {
-  color: var(--text-primary);
-}
-
```

Comment:
Not sure why we even had these, but would be great if you can go through all the files and remove all CSS code related to tooltips that we might not need anymore (including CSS variables) @nadijagraca